Kitodo/src/main/java/org/kitodo/production/helper/tasks/ExportSerialBatchTask.java

```diff
@@ -80,7 +80,7 @@
      *            batch holding a serial publication
      */
     public ExportSerialBatchTask(Batch batch) {
-        super(batch.getLabel());
+        super(ServiceManager.getBatchService().getLabel(batch));
```

Comment:
getLabel is not using the label you created

Reply:
Yeah, it is different label. The one which overwrites toString is used only for frontend.